Inventory/Items/AppleItem.gd

```diff
@@ -6,6 +6,9 @@ func _init():
 
 # apple is illegal only temporally as a test item
 func getVisibleName():
+	if(GM.pc != null):
+		if(GM.pc.hasPerk(Perk.FertilityProudMom) && getWearer() == GM.pc):
+			return "Tasty Apple"
```

Comment:
I'd rather just create a separate apple item honestly x3. Because right now this perk 'legalizes' apple stealing x3. Or you can add a variable 'notStolen' and set it to true for the perk apples.

Reply:
You still can be assaulted by the guards in the process of stealing like normal. I was thinking about making a separate "legal" apples, but that made no sense to me because stolen or not apples look the same. If you are still concerned I can make Nurse give the player apple juice instead.